src/lib/types/move/module.ts

```diff
@@ -12,3 +12,15 @@ export interface ModuleHistory {
   timestamp: Date;
   previousPolicy: Option<UpgradePolicy>;
 }
+
+/* recent modules */
+export interface RecentModule {
+  address: string;
+  block: {
+    height: number;
+    timestamp: string;
+  };
```

Comment:
we did some change to the API, and flatten the `block` field to `height` and `lastUpdated`